partition_processor.go

```diff
@@ -232,7 +236,7 @@ func (pp *PartitionProcessor) Setup(ctx context.Context) error {
 	pp.runnerGroup.Go(func() error {
 		err := pp.run(runnerCtx)
 		if err != nil {
-			pp.log.Printf("Run failed with error: %v", err)
+			pp.log.Debugf("Run failed with error: %v", err)
```

Comment:
Does it make sense to have debug logs in a library like goka?

Reply:
I was also wondering that.
The logs are mainly to understand what is going on in goka, when it's doing a rebalance etc. 
 Sarama also has internal logs that might give the user an insight why something behaves like it behaves, e.g. a rebalance because of a slow consumption - which is not necessarily an error.
For cases like this logs makes sense I think, but for the one above, probably not because this error should be logged by the client. I didn't want to remove it so I converted it to debug so we usually don't spam the user.